mu/interface.py

```diff
@@ -865,17 +878,20 @@ def dropEvent(self, event):
         source = event.source()
         self.disable(source)
         if isinstance(source, LocalFileList):
-            local_filename = os.path.join(self.home,
-                                          source.currentItem().text())
-            logger.info("Putting {}".format(local_filename))
-            try:
-                with microfs.get_serial() as serial:
-                    logger.info(serial.port)
-                    microfs.put(serial, local_filename)
-                super().dropEvent(event)
-            except Exception as ex:
-                logger.error(ex)
+            if self.findItems(source.currentItem().text(), Qt.MatchExactly) \
```

Comment:
I haven't tested this build, but if we are dragging a new file, woudn't then `self.findItems()` evaluate False and not transfer the file? (same applies to `LocalFileList.contextMenuEvent()`)